### DISCUSSION TITLE
zod-validation-error equivalent?

### DISCUSSION THREAD

**Discussion by Everduin94** [open-circle/valibot#361] [General]
Hey there!

I have a small side project that Valibot would fit really well (Zod is currently 50% of my project size) 
- https://bundlephobia.com/package/better-commits@1.8.0

I've been reading about Valibot to replace this zod pipeline. AFAIK Valibot has everything I need to do so.
- https://github.com/Everduin94/better-commits/blob/main/src/zod-state.ts

One thing I can't find. I use this library, [zod-validation-error](https://www.npmjs.com/package/zod-validation-error), to transform a zod error into a formatted error message .  

**Example:** *Validation error: Number must be greater than 0 at "id"; Invalid email at "email"*. 

Does Valibot have something that would be the equivalent?

**Comment by fabian-hiller** [comment]
Unfortunately, no. We should add such a formatting function. Feel free to create a issue or PR. Until this is implemented, you can format the Valibot issues yourself: https://valibot.dev/guides/issues/

**Reply by Everduin94** [reply]
No worries! Just wanted to make sure I wasn't missing something.

Thanks for the fast response! Love the project!
